rules/build_sector.smk

```diff
@@ -933,6 +958,7 @@ rule build_industrial_energy_demand_per_node:
         industrial_energy_demand_per_node_today=resources(
             "industrial_energy_demand_today_base_s_{clusters}.csv"
         ),
+        wallon_demands = resources("pypsa_demands_{planning_horizons}.csv"),
```

Comment:
why are pypsa and walloon_demands the same? maybe I miss the point why pypsa demands are fixed to 2050?